object_detection/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM nvcr.io/nvidia/pytorch:19.01-py3
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# install basics
+RUN apt-get update -y \
+ && apt-get install -y apt-utils git curl ca-certificates bzip2 cmake tree htop bmon iotop g++ \
+ && apt-get install -y libglib2.0-0 libsm6 libxext6 libxrender-dev
+
+# Install Miniconda
+RUN curl -so /miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
```

Comment:
Does it matter if Miniconda updates?

Reply:
Good point, Daniel. The latest submitted commit b3588e1 should address your concerns about reproducibility.